### PR TITLE
feat: add the hive doc

### PR DESCRIPTION
**Pull request by LeoArurault**

### CODE REVIEW

File: docs/integrations/thehive.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# The Hive
+
+## Overview
+
+[TheHive](https://thehive-project.org/) is a scalable, open source and free Security Incident Response Platform. This integration through a Symphony module dedicated allows you to duplicate SEKOIA.IO alerts in an instance of The Hive.
```
Comment (PierrickV) [inline docs/integrations/thehive.md:5]:
```suggestion
[TheHive](https://thehive-project.org/) is a scalable, open source and free Security Incident Response Platform. This integration through a dedicated Symphony module allows you to duplicate SEKOIA.IO alerts in an instance of The Hive.
```

File: docs/integrations/thehive.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Configure
+
+First of all, you will have to retrieve your organisation name from your The Hive instance (or [Create an organisation - The Hive documentation](https://docs.thehive-project.org/thehive/user-guides/quick-start/#create-an-organisation)). You will also need a [user apikey](https://docs.thehive-project.org/thehive/user-guides/organisation-managers/users-management/) in this organisation with `manageAlert` permission (for example with the `analyst` [profile](https://docs.thehive-project.org/thehive/user-guides/administrators/profiles/). You have now all informations to configure The Hive module and its `Create Alert` action in SEKOIA.IO.
```
Comment (PierrickV) [inline docs/integrations/thehive.md:9]:
```suggestion
First of all, you will have to retrieve your organisation name from your The Hive instance or create an organisation (see [The Hive documentation](https://docs.thehive-project.org/thehive/user-guides/quick-start/#create-an-organisation) to learn how to create an organisation).

Next, you will need a [user apikey](https://docs.thehive-project.org/thehive/user-guides/organisation-managers/users-management/) in this organisation with `manageAlert` permission (for example with the `analyst` profile [profile](https://docs.thehive-project.org/thehive/user-guides/administrators/profiles/). 

You now have all informations to configure The Hive module and its `Create Alert` action in SEKOIA.IO.
```

File: docs/integrations/thehive.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+The Hive playbook example templates are available in SEKOIA.IO : `Create a new playbook` > `Use a template` > Search for TheHive.
```
Comment (PierrickV) [inline docs/integrations/thehive.md:11]:
```suggestion
To start your playbook with an example template, go to : `Create a new playbook` > `Use a template` > Search for TheHive.
```

File: docs/integrations/thehive.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Or you can create your own on the same basis. A typical playbook to send alerts to a TheHive instance is this kind of chain :
```
Comment (PierrickV) [inline docs/integrations/thehive.md:13]:
```suggestion
You can also create your own on the same basis. A typical playbook to send alerts to a TheHive instance will use this kind of chain :
```

**Review by cdum0nt — DISMISSED** [review]

File: mkdocs.yml [hunk 5]
```diff
@@ -89,6 +89,8 @@ nav:
       - Common Event Format: integrations/cef.md
     - SEKOIA.IO:
       - Activity logs: integrations/sekoiaio_activity_logs.md
+    - External tools:
```
Comment (squioc) [inline mkdocs.yml:92]: Should we not consider The hive as an Application (instead of an External tools)?

File: docs/integrations/thehive.md (hunk 1 above)
Comment (squioc) [inline docs/integrations/thehive.md:5]: Suggestion enhancement:
```suggestion
[TheHive](https://thehive-project.org/) is a scalable, open source and free Security Incident Response Platform. This integration allows you to forward SEKOIA.IO alerts in an instance of The Hive.
```

**Review by squioc — APPROVED** [review]

File: docs/integrations/thehive.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+## Configure
+
+First of all, you will have to retrieve your organisation name from your The Hive instance or create an organisation (see [The Hive documentation](https://docs.thehive-project.org/thehive/user-guides/quick-start/#create-an-organisation) to learn how to create an organisation).
+
+Next, you will need a [user apikey](https://docs.thehive-project.org/thehive/user-guides/organisation-managers/users-management/) in this organisation with `manageAlert` permission (for example with the `analyst` profile [profile](https://docs.thehive-project.org/thehive/user-guides/administrators/profiles/). 
+
+You now have all informations to configure The Hive module and its `Create Alert` action in SEKOIA.IO.
+
+To start your playbook with an example template, go to : `Create a new playbook` > `Use a template` > Search for TheHive.
```
Comment (cdum0nt) [inline docs/integrations/thehive.md:15]:
```suggestion
To start your playbook with an example template, go to : "Create a new playbook" > "Use a template" > Search for TheHive.
```

**Review by cdum0nt — DISMISSED** [review]

File: docs/integrations/thehive.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+First of all, you will have to retrieve your organisation name from your The Hive instance or create an organisation (see [The Hive documentation](https://docs.thehive-project.org/thehive/user-guides/quick-start/#create-an-organisation) to learn how to create an organisation).
+
+Next, you will need a [user apikey](https://docs.thehive-project.org/thehive/user-guides/organisation-managers/users-management/) in this organisation with `manageAlert` permission (for example with the `analyst` profile [profile](https://docs.thehive-project.org/thehive/user-guides/administrators/profiles/). 
+
+You now have all informations to configure The Hive module and its `Create Alert` action in SEKOIA.IO.
+
+To start your playbook with an example template, go to : "Create a new playbook" > "Use a template" > Search for TheHive.
+
+You can also create your own on the same basis. A typical playbook to send alerts to a TheHive instance will use this kind of chain :
```
Comment (squioc) [inline docs/integrations/thehive.md:17]: I don't know if it's an omission or a choice, but to be consistent with others occurrences of the name of the product:
```suggestion
You can also create your own on the same basis. A typical playbook to send alerts to a The Hive instance will use this kind of chain :
```

**Pull request merged**